packages/svelte/tests/animation-helpers.js

```diff
@@ -62,6 +63,10 @@ class Animation {
 				return {
 					/** @param {() => void} callback */
 					catch: (callback) => {
+						// @ts-ignore
+						this.currentTime = null;
+						// @ts-ignore
+						this.startTime = null;
```

Comment:
shouldn't this go in the `cancel` method below?

Reply:
Yep, done!